fabric_plugin/tasks.py

```diff
@@ -70,6 +70,25 @@ def _load_private_key(key_contents):
     )
 
 
+def _check_python_version(run_func):
+    def _which_python():
+        try:
+            # This is usually reference to python2 and it could reference to
+            # python3 if `alternatives mechanism used to enable the
+            # unversioned python`
+            py_version = run_func(
+                'python -c \'import sys; print(sys.version_info[0])\''
+            ).stdout
+        except Exception:
+            py_version = '3'
+            py_path = run_func('which python3').stdout.rstrip()
```

Comment:
you're already doing `which` so what's the reason to have the `python -c` too?

Reply:
Remove the whole method, I'm depending on source right now